shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -144,12 +153,21 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
 
   const DeviceProfile device_profile;
 
+  FLUTTER_API_SYMBOL(FlutterEngine) Engine() { return engine_; }
```

Comment:
You don't need this method and I think if you need this, I think something is going wrong.

Reply:
Oh.. got it.